pybaseball/league_pitching_stats.py

```diff
@@ -1,9 +1,13 @@
-import requests
-import pandas as pd
-import numpy as np
+import datetime
 import io
+
+import numpy as np
+import pandas as pd
+import requests
 from bs4 import BeautifulSoup
-import datetime
+
+from .datahelpers import caching
+
```

Comment:
Do you have to reorder these all the time? Adds to the diff and not a really good reason for it

Reply:
I reorder them in the standard python ordering. From PEP8 (https://www.python.org/dev/peps/pep-0008/#imports)

Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.

Reply:
Good call, if you're already adding something there.